.travis.yml

```diff
@@ -2,7 +2,7 @@ language: python
 python:
   - "3.6"
 env:
-  - ES_VERSION=5.6.13 ES_DOWNLOAD_URL=https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-${ES_VERSION}.tar.gz
+  - ES_VERSION=6.7.0 ES_DOWNLOAD_URL=https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-${ES_VERSION}.tar.gz
```

Comment:
Aha - we can do multiple env builds for Travis. The example here is for `DB` but don't see why it can't work for `ES_VERSION`: https://docs.travis-ci.com/user/database-setup/#multiple-database-builds

Reply:
Okay, got unit tests working across 5 and 6, trying out a Travis build matrix now!

Reply:
It worked! :tada:

Reply:
Fab 🏆